File: Week6/Problem/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int


def main():
    # Get the height of the pyramid
    H = get_height()
    # Loop through the height
    for height in range(H):
        # Create variables for both sections
        spaces = (H - height - 1) * " "
        hashes = (height + 1) * "#"
        # Finally print the level
        print(spaces + hashes)


# Create the function to get the height
def get_height():
    # Check an infinte loop
    while True:
        # Force a correct answer
        height = get_int("Height: ")
        if height > 0 and height < 9:
            # Escape the loop
            return height


# Call the main function
main()

File: Week1/Problem/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // ask the user for the pyramid's height
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0 || height >= 9);

    // start the loop for the pyramid
    for (int level = 1; level <= height; level++)
    {
        int spaces = height - level;

        // Print spaces
        for (int i = 0; i < spaces; i++)
        {
            printf(" ");
        }

        // Print left hashes
        for (int i = 0; i < level; i++)
        {
            printf("#");
        }

        // Print gap
        printf("  ");

        // Print right hashes
        for (int i = 0; i < level; i++)
        {
            printf("#");
        }

        printf("\n");
    }
}